CORE/spec/api_engine_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rspec'
 require 'httpi'
-require_relative '../modules/sql_engine/api_engine'
+require_relative '../modules/api_engine/api_engine.rb'
```

Comment:
REAAAAAALY nitpicky, but i have the CORE directory set as a library under test in the rspec defintion (see loadpath expansion in spechelper)

Simplest way to require anything in the CORE directory is as such, `require 'modules/api_engine/api_engine'`

That said, i think i'm going to do away with the subdirectories in the modules directory for api and sql engine in a changeset soon. because neither is complicated enough to need a namespace. (if i pull in sms or email client they would need one for example)